app/src/main/java/com/example/habitsmasher/Habit.java

```diff
@@ -0,0 +1,67 @@
+package com.example.habitsmasher;
+
+import java.util.Date;
+
+/**
+ * This is the Habit class
+ * Its purpose is to store and retrieve the title, reason, and date of a given habit
+ */
+public class Habit {
+    private String _title;
+    private String _reason;
+    private Date _date;
+
+    /**
+     *
+     * @return _title : The title of the habit
+     */
+    public String get_title() {
```

Comment:
- [ ] Make all getters/setters in this class camelCase please

Reply:
Done.